rails/client/.eslintrc

```diff
@@ -11,6 +11,8 @@
   },
   "rules": {
     "max-len": "off", // Remove warnings on max line length exceeding 100 characters
+    "object-curly-newline": "off",
```

Comment:
you should just fix these :P

Reply:
I ran eslint --fix to resolve the object-curly-newline. I'd like to update the destructuring rule on a different pull request in the future because it requires code changes to deconstruct and alias the new variables.